### PR TITLE
Update packs/magento.json

### PR DESCRIPTION
**Pull request by DrewML**
**Notes**

**offscreen-images**
Magento's biggest client-side bottleneck is JavaScript (shocker I know), so opted to suggest the progressive enhancement route instead of a JS-based solution. This work is currently being done by a team in the core, so we'll hopefully update this soon to just verify the feature is enabled.

**uses-rel-preconnect/uses-rel-preload**
Preferable for us to have folks change things in a way that's backed by a VCS (admin side-steps this)

**critical-request-chains**
Any Magento store not bundled (default) is going to trigger this one for the JS, so I opted to be specific here. Won't address long chains from other content types, but JS is the common one for us

**unused_css_rules**
Some work will be in a newer version of Magento to ease this (a tiny bit). Will circle back with another PR when that's in a general release

### CODE REVIEW

**Comment by DrewML** [comment]
@googlebot I signed it!

File: packs/magento.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-  "unminified_css": "",
-  "unminified_javascript": "",
+  "offscreen_images": "Consider modifying your product and catalog templates to make use of the web platform's [lazy loading](https://web.dev/native-lazy-loading) feature",
+  "total_byte_weight": "Disable Magento's built-in JavaScript bundling and minification, and consider using [baler](https://github.com/magento/baler/)",
+  "render_blocking_resources": "Disable Magento's built-in JavaScript bundling, and consider using [baler](https://github.com/magento/baler/) instead",
+  "unminified_css": "Enable the \"Minify CSS Files\" option in the store's Developer settings",
+  "unminified_javascript": "Use [Terser](https://www.npmjs.com/package/terser) to minify all JavaScript assets outfrom from static content deployment, and disable the built-in minification feature",
   "efficient_animated_content": "",
-  "unused_javascript": "",
+  "unused_javascript": "Disable Magento's built-in JavaScript bundling",
```
Comment (housseindjirdeh) [inline packs/magento.json]: Any chance we have a link to a doc page/instructions for this?

Comment (housseindjirdeh) [reply packs/magento.json]: If not no worries, I'll link to [this](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/js-bundling.html) if there's no specific page that describes disabling

**Comment by housseindjirdeh** [comment]
So great, thanks so much for taking a look and dropping these suggestions so last minute 🎉 

Excited to see this rolling out :D

File: packs/magento.json (hunk 1 above)
Comment (DrewML) [reply packs/magento.json]: Nothing at the moment, but the plan is to get these features ripped out in a major release (they're old and counter productive)

**Pull request merged**